ocs_ci/ocs/scale_lib.py

```diff
@@ -242,6 +248,15 @@ def create_scale_pods(
                         "Scaling PVC+POD failed analyze setup and log for more details"
                     )
 
+    def pvc_expansion(self, pvc_new_size):
+        """
+        Function to expand PVC size and verify the new size is reflected.
+        """
+        logging.info(f"PVC size is expanding to {pvc_new_size}")
+        for pvc_oj in self.pvc_obj:
```

Comment:
did you mean `pvc_obj`?

Reply:
It is pvc in pvc_obj.  I'm changing to pvc_object instead.